server.cpp

```diff
@@ -160,7 +264,7 @@ int main(int argc, char* argv[])
 
             const std::string date = make_daytime_string();
```

Comment:
`date` is only used in the `write_log`. Let's use it there directly, rather then passing through the param.